pkg/volume/local_volume/local_volume.go

```diff
@@ -251,10 +280,24 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 
-	// TODO: how to prevent multiple mounts with conflicting fsGroup?
 	return volume.SetVolumeOwnership(m, fsGroup)
 }
 
+// isBindMounted checks whether the volume has beend mounted before.
+func (m *localVolumeMounter) isBindMounted() (bool, error) {
+	mountpoints, err := m.mounter.List()
+	if err != nil {
+		return false, err
+	}
+	for _, mountpoint := range mountpoints {
```

Comment:
@msau42 

While looking at SELinux issue, I find that kubelet does have a cache, I'll take a look if that might be helpful.

https://github.com/kubernetes/kubernetes/blob/3fbfafdd0a86c2a6fb4f53d94e47ba9592e19601/pkg/kubelet/container/runtime.go#L447